json_schemas/users.json

```diff
@@ -22,6 +22,11 @@
       "type": "string",
       "pattern": "^(buyer|supplier|admin)$"
     },
+    "supplier_id": {
+      "type": "string",
```

Comment:
And I think for now the JSON should match the database - either way is fine, but a mix of number and string seems definitely wrong.